src/pages/dashboardTab/components/tabView.js

```diff
@@ -73,6 +73,21 @@ const TabView = ({
   const AddToCartAlertDialogRef = useRef();
 
   const [cartIsFull, setCartIsFull] = React.useState(false);
+
+  const getUpdatedColumns = () => {
+    const updateColumns = getColumns(customColumn, classes, data, externalLinkIcon);
+    const disableViewColumnsList = customColumn.columns
+      .filter((c) => c.viewColumns !== undefined && !c.viewColumns);
```

Comment:
a small one, can you make iterator more readable

Reply:
update

Reply:
filter((c) => c.viewColumns !== undefined && !c.viewColumns);

Some thing more informative name rather than just c will help for readability of the code

Reply:
updated with proper naming convention